cores/esp8266/Esp.cpp

```diff
@@ -579,24 +579,25 @@ bool EspClass::flashEraseSector(uint32_t sector) {
 #if PUYA_SUPPORT
 static int spi_flash_write_puya(uint32_t offset, uint32_t *data, size_t size) {
     if (data == nullptr) {
-      return 1; // SPI_FLASH_RESULT_ERR
+      return SPI_FLASH_RESULT_ERR; // SPI_FLASH_RESULT_ERR
```

Comment:
can you remove the now redundant comment?

Reply:
Done